src/model/application/command_interaction.rs

```diff
@@ -74,13 +76,21 @@ pub struct CommandInteraction {
     /// Always `1`.
     pub version: u8,
     /// Permissions the app or bot has within the channel the interaction was sent from.
+    // TODO(next): This is now always serialized.
     pub app_permissions: Option<Permissions>,
     /// The selected language of the invoking user.
     pub locale: String,
     /// The guild's preferred locale.
     pub guild_locale: Option<String>,
     /// For monetized applications, any entitlements of the invoking user.
     pub entitlements: Vec<Entitlement>,
+    #[cfg(feature = "unstable_discord_api")]
+    #[serde(rename = "authorizing_integration_owners")]
+    /// The owner of the application that authorized the interaction, such as a guild or user.
+    pub authorizing_integration_owner: AuthorizingIntegrationOwner,
```

Comment:
did this change get reverted due to force pushing?
```suggestion
    pub authorizing_integration_owner: Option<AuthorizingIntegrationOwner>,
```